turborepo-tests/helpers/find_global_hash.sh

```diff
@@ -6,7 +6,7 @@
 # 2023-04-06T04:28:19.599Z [DEBUG] turbo: global hash: value=a027dadc4dea675e
 #
 # Usage:
-# turbo build -vv 2>&1 | "$TESTDIR/./get-global-hash.sh"
+# turbo build -vv 2>&1 | "$TESTDIR/./find_global_hash.sh"
```

Comment:
Do I feel strongly about this? No. But I like how we have everything segmented in the global helpers directory by prefix:

`setup_*`
`mock_*`
`utils_*`

So wanted to generalize this. But not locked in.